subsys/net/lib/nrf70_fw_ext/ext_fw_load.c

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief File containing FW load functions for Zephyr.
+ */
+#include <stdlib.h>
+#include <string.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP) && defined(CONFIG_NORDIC_QSPI_NOR)
+#include <zephyr/drivers/flash/nrf_qspi_nor.h>
+#endif /* CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP */
+
+#include <zephyr/logging/log.h>
+#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP) && defined(CONFIG_NORDIC_QSPI_NOR)
+/* For NRF QSPI NOR special handling is needed for this file as all RODATA of
+ * this file is stored in external flash, so, any use of RODATA has to be protected
+ * by disabling XIP and enabling it again after use. This means no LOG_* macros
+ * (buffered) or buffered printk can be used in this file, else it will crash.
+ */
+LOG_MODULE_DECLARE(wifi_nrf, LOG_LEVEL_NONE);
+#else
+LOG_MODULE_DECLARE(wifi_nrf, CONFIG_WIFI_NRF70_LOG_LEVEL);
+#endif /* CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP */
+
+#include <fmac_main.h>
+
+#ifdef CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP
+/* INCBIN macro Taken from https://gist.github.com/mmozeiko/ed9655cf50341553d282 */
+#define STR2(x) #x
+#define STR(x) STR2(x)
```

Comment:
use `STRINGIFY` from Zephyr

Reply:
Just had a chat with Jamie while trying to fix another problem, will remove INCBIN and use `generate_inc_file_for_target` to add proper dependencies.